src/blocks/utils/variable.ts

```diff
@@ -76,8 +76,7 @@ export function varNameForType(type: string): string {
   }
   if (type.includes('[')) {
     // The type is an array.
-    // TODO(lizlooney): What's a good name for an array variable?
-    return '';
+    return 'myArray';
```

Comment:
Does this need to be localized?  (or does that create a ton of problems)

Reply:
I'm not sure. I've added a TODO.